source/nl/1.0.0/index.html.haml

```diff
@@ -0,0 +1,299 @@
+
+  %h4#principles
+    %a.anchor{ href: "#principles", aria_hidden: "true" }
+    Richtlijnen
+
+  %ul
+    %li
+      Changelogs zijn <em>voor mensen</em>, niet voor machines.
+    %li
+      Er zou een vermelding moeten zijn voor elke versie.
+      There should be an entry for every single version.
+    %li
+      Aanpassingen van het zelfde type moeten gegroepeerd worden.
+    %li
+      Versies en secties souden linkbaar moeten zijn.
+    %li
+      De laatste versie staat bovenaan.
+    %li
+      De release datum van elke versie word weergegeven.
+    %li
+      Geef aan of je #{link_to "Semantic Versioning", semver} gebruikt.
+
+  %a.anchor{ href: "#types", aria_hidden: "true" }
+  %h4#types Types of changes
+
+  %ul
+    %li
+      %code Added
+      voor nieuwe functionaliteit.
+    %li
+      %code Changed
+      voor aanpassingen aan bestaande functionaliteit.
+    %li
+      %code Deprecated
+      voor functionaliteit die binnenkort komt te vervallen.
+    %li
+      %code Removed
+      voor functionaliteit die vanaf nu vervallen is.
+    %li
+      %code Fixed
+      voor bug fixes.
+    %li
+      %code Security
+      voor aanpassingen met betrekking tot veiligheid.
+
+.effort
+
+  %h3#effort
+    %a.anchor{ href: "#effort", aria_hidden: "true" }
+    Hoe kan ik zo min mogelijk moeite stoppen in het bijhouden van een changelog?
+
+  %p
+    Houd bovenin een <code>Unreleased</code> sectie bij met aanpassingen voor de komende release.
+
+  %p Dit heeft twee doelen:
+
+  %ul
+    %li
+      Mensen kunnen zien wat ze kunnen verwachten in aankomende releases.
+    %li
+      Als je een release doet kan je eenvoudig de <code>Unreleased</code> sectie
+      aanpassen naar een nieuwe release sectie.
+
+.bad-practices
+  %h3#bad-practices
+    %a.anchor{ href: "#bad-practices", aria_hidden: "true" }
+    Kan een changelog slecht zijn?
+
+  %p Ja. Hier een paar manieren waarop je een changelog behoorlijk onbruikbaar kan maken.
+
+  %h4#log-diffs
+    %a.anchor{ href: "#log-diffs", aria_hidden: "true" }
+    Commit log diffs
+
+  %p
+    Commit log diffs gebruiken als een changelog is een slecht idee:
+    ze staan vol met ruis. Denk bijvoorbeeld aan merge commits, commits met
+    onduidelijke titels, documentatie aanpassingen etc.
+
+  %p
+    Het doel van een commit bericht is om één enkele stap in de evolutie van de
+    code te beschrijven.
+
+  %p
+    Het doel van een changelog is om noemenswaardige aanpassingen te documenteren,
+    vaak over meerdere commits, en om deze duidelijk naar de eindgebruiker te communiceren.
+
+  %h4#ignoring-deprecations
+    %a.anchor{ href: "#ignoring-deprecations", aria_hidden: "true" }
+    Deprecations negeren
+
+  %p
+    Wanneer mensen upgraden van de ene naar de andere versie,
+    moet het overduidelijk zijn als er iets niet meer zal werken.
+    Het moet mogelijk zijn om te upgraden naar een versie met deprications,
+    afhanlekijkheden van de deprications weg te halen, en vervolgens
```

Comment:
type: afhanlekijkheden  => afhankelijkheden
suggestion: "afhankelijkheden van de deprications weg te halen" => "de deprications weg te halen"

Reply:
In je aanpassing heb je het veranderd naar "_vervolgens de deprications weg te halen, en vervolgens de upgrade..._", naar mijn mening klinkt het dubbele gebruik van "vervolgens" niet heel goed 🤔 misschien dat "_vervolgens de deprications weg te halen, en daarna de upgrade..._" een beter alternatief is?